src/web/components/elements/RichLinkComponent.tsx

```diff
@@ -23,6 +23,24 @@ interface CAPIRichLinkType {
 	format: CAPIFormat;
 	starRating?: number;
 	contributorImage?: string;
+	imageAsset: ImageAsset;
+}
+interface ImageAsset {
+	index: number;
+	fields: ImageAssetFields;
+	mediaType: string;
+	url: string;
+}
+interface ImageAssetFields {
+	displayCredit: string;
+	source: string;
+	photographer: string;
+	isMaster: string;
+	altText: string;
+	height: string;
+	credit: string;
+	mediaId: string;
+	width: string;
 }
```

Comment:
What do we use these extra fields for (isMaster, source, etc)? Can we remove them from the interface?

Reply:
They aren't being used, but I thought it might be good to expose what is available in the data in case we do decide to use them in the future